openstack/placement/templates/_helpers.tpl

```diff
@@ -9,10 +9,5 @@
 
 
 {{- define "placement.db_name" }}
-  {{- if or (eq .Values.dbType "mariadb") (eq .Values.dbType "pxc-db") }}
 {{- include "utils.db_host" . -}}
-{{/* Use nova-api-mariadb database, if no database is selected explicitly */}}
```

Comment:
This logic was added to allow for the following configuration:
`mariadb.enabled` set to `true`, `mariadb.name` set to `placement` - new empty database is started, and dbType="", so placement continues to use `nova-api-mariadb` database until explicitly migrated